application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/ECChronos.java

```diff
@@ -97,9 +100,19 @@ public ECChronos(Properties configuration, RepairFaultReporter repairFaultReport
                 .withDefaultRepairConfiguration(getRepairConfiguration(repairProperties))
                 .build();
 
+        ReplicationState replicationState = new ReplicationState(metadata, host);
+        myOnDemandRepairScheduler = OnDemandRepairSchedulerImpl.builder()
+                .withScheduleManager(myECChronosInternals.getScheduleManager())
+                .withTableRepairMetrics(myECChronosInternals.getTableRepairMetrics())
+                .withJmxProxyFactory(myECChronosInternals.getJmxProxyFactory())
+                .withReplicationState(replicationState)
+                .withRepairLockType(repairProperties.getRepairLockType())
+                .withMetadata(metadata)
```

Comment:
Should we pass the default repair configuration to the repair scheduler to honor the configurations done for ecc? (unwind time, etc)

Reply:
There is an argument to keep those configurations separate, but we decided to use the configurations for ecc